app/javascript/psulib_blacklight/javascripts/availability.js

```diff
@@ -109,47 +117,68 @@ function loadAvailability(locations) {
                 // Now that the availability data has been rendered, check for ILL options and update links
                 $('.availability-holdings [data-type="ill-link"]').each(function() {
                     var catkey = $(this).data('catkey');
-                    createILLURL($(this), catkey)
+                    var callNumber = $(this).data('call-number');
+                    var archivalThesis = $(this).is('[data-archival-thesis]');
+                    var item = {
+                        catkey: catkey,
+                        callNumber: callNumber,
+                        archivalThesis: archivalThesis
```

Comment:
Did you swap the `item` object's contents? ILL URL has `archivalThesis`. Seems like that should be handed to createAeonURL. ?

Reply:
nvm, I see that it is needed.